<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Download</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select Resolution</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>144p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>240p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>360p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>720p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1080p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1440p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2160p</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
